Ten Elephants/Ten Elephants/Views/WideCellView.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  WideCellView.swift
+//  Ten Elephants
+//
+//  Created by Kirill Denisov on 10.12.2021.
+//
+
+import UIKit
+
+class WideCellView: UICollectionViewCell {
+
+    private enum Constants {
+        static let labelGap: CGFloat = 10
+        static let labelHeight: CGFloat = 20
+        static let cornerRadius: CGFloat = 20
+        static let titleFontSize: CGFloat = 20
+        static let subtitleFontSize: CGFloat = 16
+        static let imageBottomGap: CGFloat = -60
+    }
+
+    lazy var titleLabel: UILabel = {
```

Comment:
Давай повыносим создание куда-нибудь в эксстеншен ниже, чтобы тут был почише код
```suggestion
    lazy var titleLabel = makeTitleLabel()
```

```swift
extension WideCellView {
  private func makeTitleLabel() -> UILabel { ... }
}
```

Reply:
И для другие таких ui пропертей